glib/src/param_spec.rs

```diff
@@ -475,14 +481,14 @@ macro_rules! define_builder {
             $($field_id: Option<$field_ty>),*
         }
         impl<'a> $builder_type<'a> {
-            /// Default: `self.name`
-            pub fn nick(mut self, nick: &'a str) -> Self {
```

Comment:
Why would you pass an Option<T> to a builder type.. This change doesn't make sense

Reply:
Fixed